src/update.user.request.ts

```diff
@@ -0,0 +1,12 @@
+import { IsString, IsNumber } from "class-validator";
+
+export class UpdateUserRequest {
+    @IsNumber()
+    public id: number;
```

Comment:
shouln't be private? Does this work with swagger?

Reply:
The other classes have the public properties too...

Reply:
They were public because they work with swagger...
So two thinks we can do:
1. Try private in both classes and see if we cal call the endpoints and if the swagger generated documentation works good.
2. Keep them as public and then create another issue to solve this.

Reply:
Option 2.